src/client/contextmenu.js

```diff
@@ -68,11 +68,14 @@ export default class ContextMenu {
       }, "CMD+K"],
       ["Browser",     (evt) => {
         this.openComponentInWindow("lively-container", evt).then(comp => {
-          var container = _.last(document.querySelectorAll("lively-container"));
+          let containers = Array.from(document.querySelectorAll('lively-container'))
```

Comment:
the `last` function is much more intention revealing than `slice(-1)[0]` I would rather define them myself in `Array` ?